src/components/Dashboard/Dashboard.jsx

```diff
@@ -31,12 +34,24 @@ export const Dashboard = props => {
     props.getTimeZoneAPIKey();
   }, []);
 
-  useEffect(() => {
-    if (props.match.params && props.match.params.userId && userId != props.match.params.userId) {
-      userId = props.match.params.userId;
-      getUserProfile(userId);
-    }
-  }, [props.match]);
+  useEffect(()=>{
+  },[userProfile])
```

Comment:
What is the purpose of this useEffect hook here?

Reply:
Hello, @stevenpaalz, thank you, I forgot to remove that.